consoles/node-network.html

```diff
@@ -0,0 +1,61 @@
+{{ template "head" . }}
+
+{{ template "prom_right_table_head" }}
+  <th colspan="2">Interface</th>
+</tr>
+{{ range printf "node_network_transmit_bytes{job='node',instance='%s'}" .Params.instance | query | sortByLabel "device" }}
+  <th colspan="2">{{ .Labels.device }}</th>
+  <tr>
+    <td>Transmit Rate</td>
+    <td>{{ template "prom_query_drilldown" (args (printf "rate(node_network_transmit_bytes{job='node',instance='%s',device='%s'}[5m])" .Labels.instance .Labels.device) "B/s" "humanize1024") }}</td>
```

Comment:
I think for default consoles we'd have NFS on it's own page once we had those stats in the node_exporter, and we'd look to make whatever you find useful from collectd available via the node exporter.

We do have all of `netstat -s` available by default, there's probably some interesting stats in there.